javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,30 @@
+//save a note
+const notes=[];
+function saveNote(content, id) {
+    const resultNote={content:content, id:id};
```

Comment:
A shortcut for this is.
```suggestion
    const resultNote={content, id};
```

Reply:
Modified the code by giving the shortcut